lib/text_handler.rb

```diff
@@ -0,0 +1,35 @@
+require 'dotenv'
+Dotenv.load('./twilio_credentials.env')
+require 'twilio-ruby'
+
+class TextHandler
+  def initialize
+    read_env_variables
+  end
+
+  def send_confirmation_text(order_number, text_client = Twilio::REST::Client)
+    client = text_client.new(@account_sid, @auth_token)
+    begin
+      client.messages.create(
+        from: @takeaway_number,
+        to: order_number,
+        body: create_message_body
+      )
+    rescue RuntimeError => e
+      raise RuntimeError, "unable to send text message. Error from text client - #{e.message}"
```

Comment:
I like that you caught the error and decorated it with useful context. In this case we're catching and throwing it again, which adds a runtime overhead for these errors. Might the stack trace be enough to understand this error without catching it?